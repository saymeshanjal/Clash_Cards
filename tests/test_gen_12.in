51
14 91 3
4 12 2
6 83 1
6 86 5
10 92 3
10 45 2
4 88 5
6 78 3
8 22 1
12 61 4
12 87 5
8 1 4
12 87 5
8 45 3
12 45 3
4 14 3
10 19 4
10 5 4
2 20 1
16 5 3
6 58 5
2 54 3
16 23 2
12 62 2
4 19 5
6 8 4
12 28 5
2 37 5
12 97 1
4 25 3
6 75 2
14 40 3
2 87 1
2 48 5
6 13 4
4 76 2
6 33 2
14 73 1
16 26 1
4 67 1
4 5 1
8 80 3
8 44 5
14 32 5
16 94 4
8 75 2
8 41 5
8 11 5
4 43 3
10 56 1
10 57 3