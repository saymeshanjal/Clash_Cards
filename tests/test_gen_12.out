1652 1780 0
1 2 3
10 10 0
1 19 0
110 142 0
76 77 99
0 6 0
6 14 0
35 35 0
208 441 275
219 705 0
0 70 0
219 705 0
27 43 0
424 500 0
1 5 0
3 216 33
0 246 6
1 1 0
495 12375 0
1 19 0
0 2 0
365 1590 10915
286 287 351
0 6 0
0 19 1
32 892 0
0 2 0
462 462 0
1 5 0
5 6 9
810 2622 0
1 1 0
0 2 0
0 15 5
1 2 3
5 6 9
1716 1716 0
5997 6873 0
3 3 0
3 3 0
27 43 0
8 62 0
99 3333 0
3809 5406 3655
20 21 29
8 62 0
0 70 0
1 5 0
126 126 0
110 142 0
